juju/model.py

```diff
@@ -131,13 +131,15 @@ class Relation:
     def __init__(self, relation_name, relation_id, local_unit, backend, cache):
         self.name = relation_name
         self.id = relation_id
-        self.apps = set()
+        self.app = None
```

Comment:
In other words, this makes an instance of `Relation` to be a representation of a `relation-id` rather than an `endpoint` as far as I understand.

Reply:
That has always been the case with this implementation.  We don't currently have a model class for the relation endpoint, but it would have been the `RelationList` that was in there for a bit, since it holds all of the relation instances for a given endpoint name.